python/lsst/obs/monocam/monocamIsrTask.py

```diff
@@ -0,0 +1,142 @@
+#
+# LSST Data Management System
+# Copyright 2008-2016 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+import lsst.ip.isr as ip_isr
+import lsst.pipe.base as pipe_base
+
+class MonocamIsrTask(ip_isr.IsrTask):
+    @pipe_base.timeMethod
+    def run(self, ccdExposure, bias=None, dark=None,  flat=None, defects=None, fringes=None, bfKernel=None):
```

Comment:
This demonstrates that the `IsrTask.run` method needs to be made more modular, to avoid copying and pasting so much.

Please indicate what's changed between the default `IsrTask.run` and this override.